networking/gcp/vpc/examples/complete/main.tf

```diff
@@ -0,0 +1,35 @@
+# random string
+resource "random_string" "suffix" {
+  length  = 5
+  special = false
+  upper   = false
+  numeric = true
+}
+
+locals {
+  cidr = "10.0.0.0/16"
+}
+
+module "complete_vpc" {
+  source                                    = "../../../vpc"
+  name                                      = "complete-${random_string.suffix.result}"
+  auto_create_subnetworks                   = false
+  routing_mode                              = "GLOBAL"
+  mtu                                       = 1460
+  enable_ula_internal_ipv6                  = false
+  internal_ipv6_range                       = null
+  network_firewall_policy_enforcement_order = "AFTER_CLASSIC_FIREWALL"
+  delete_default_routes_on_create           = true
+  shared_vpc_host_project                   = false
+  private_subnets                           = [for k in range(3) : cidrsubnet(local.cidr, 4, k)]
+  public_subnets                            = [for k in range(3) : cidrsubnet(local.cidr, 8, k + 48)]
+  gke_subnets = {
+    "gke-alpha" = {
+      nodes_cidr_block    = "10.10.0.0/8",
```

Comment:
I think you meant either `10.0.0.0/8` or more likely `10.10.0.0/16`.

Reply:
Yes, 10.10.0.0/16